pyproject.toml

```diff
@@ -0,0 +1,133 @@
+[build-system]
+requires = ["setuptools>=65.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "udata"
+version = "4.1.3.dev"
+description = "Open data portal"
+readme = "README.md"
+authors = [
+    {name = "Opendata Team", email = "opendatateam@data.gouv.fr"},
+]
+license = {text = "GNU AGPLv3+"}
+keywords = ["udata", "opendata", "portal", "data"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python",
+    "Environment :: Web Environment",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Developers",
+    "Topic :: System :: Software Distribution",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)"
+]
+dependencies = [
+    "authlib==0.14.3",
+    "awesome-slugify==1.6.5",
+    "Babel==2.8.0",
+    "bcrypt==3.1.7",
+    "bleach==3.1.0",
+    "blinker==1.4",
+    "celery==4.3.0",
+    "celerybeat-mongo==0.2.0",
+    "chardet==3.0.4",
+    "click==7.0",
+    "cryptography==2.8",
+    "factory-boy==2.12.0",
+    "Faker==3.0.0",
+    "Flask-BabelEx==0.9.3",
+    "Flask-Caching==1.8.0",
+    "Flask-CDN==1.5.3",
+    "Flask-Cors==3.0.8",
+    "flask-fs==0.6.1",
+    "Flask-Gravatar==0.5.0",
+    "Flask-Login==0.4.1",
+    "Flask-Mail==0.9.1",
+    "flask-mongoengine==0.9.5",
+    "Flask-Navigation==0.2.0",
+    "flask-restplus==0.13.0",
+    "Flask-Security-Too==3.4.4",
+    "Flask-Sitemap==0.3.0",
+    "Flask-Themes2==0.1.4",
+    "Flask-WTF==0.14.2",
+    "Flask==1.1.4",
+    "geojson==2.5.0",
+    "html2text==2019.9.26",
+    "itsdangerous==1.1.0",
+    "Jinja2==2.10.3",
+    "kombu[redis]==4.4.0",
+    "lxml==4.4.2",
+    "mistune==0.8.4",
+    "mkdocs==1.2.3",
+    "mongoengine==0.20.0",
+    "jsonschema==3.2.0",
+    "msgpack==0.6.2",
+    "netaddr==0.7.19",
+    "pillow==8.0.0",
+    "pydenticon==0.3.1",
+    "pymongo==3.10.1",
+    "python-dateutil==2.8.1",
+    "pytz==2019.3",
+    "redis==3.3.11",
+    "rdflib==6.0.0",
+    "requests==2.24.0",
+    "sentry-sdk[flask]==1.1.0",
+    "simplejson==3.17.0",
+    "StringDist==1.0.9",
+    "tlds",
+    "udata_event_service==0.0.8",
+    "ujson==1.35",
+    "urlextract==0.14.0",
+    "unicodecsv==0.14.1",
+    "vine==1.3.0",
+    "voluptuous==0.11.7",
+    "werkzeug==0.16.1",
+    "wtforms-json==0.3.3",
+    "wtforms==2.2.1",
+    "xmltodict==0.12.0"
+]
+
+[project.optional-dependencies]
+dev = [
```

Comment:
We should probably document how to install these? Or should we install based on requirements/develop.pip ?